.github/workflows/build-packages.yml

```diff
@@ -173,7 +165,7 @@ jobs:
         run: |
           for os in ${{ inputs.os }}; do
             for architecture in x86_64 ${{ vars.ARM64_USE_UBICLOUD == '1' && 'aarch64' || '' }}; do
-              [[ "$os" != "el-7" || "$architecture" != "aarch64"  ]] && echo "\"pkghashes-$os-$architecture\"" | tee -a /tmp/os-inputs.txt
```

Comment:
is the != aarch64 check gone on purpose?

Reply:
that condition is supposed to be equivalent to `| grep -v pkghashes-el-7-aarch64`, in which case it can go. If it's not equivalent, I made a mistake :)